server/interceptor/interceptor.go

```diff
@@ -248,7 +248,18 @@ type rpcInfo struct {
 // TreeType and Readonly are determined based on the request type.
 func getRPCInfo(req interface{}, quotaUser string) (*rpcInfo, error) {
```

Comment:
While you're in here, please can you fix the comment for `rpcInfo.opts`, which refers to a nonexistent field (`doesNotHaveTree`)

Reply:
Done, well spotted.